src/config.c

```diff
@@ -3061,6 +3061,7 @@ standardConfig static_configs[] = {
     createBoolConfig("latency-tracking", NULL, MODIFIABLE_CONFIG, server.latency_tracking_enabled, 1, NULL, NULL),
     createBoolConfig("aof-disable-auto-gc", NULL, MODIFIABLE_CONFIG, server.aof_disable_auto_gc, 0, NULL, updateAofAutoGCEnabled),
     createBoolConfig("replica-ignore-disk-write-errors", NULL, MODIFIABLE_CONFIG, server.repl_ignore_disk_write_error, 0, NULL, NULL),
+    createBoolConfig("hide-client-info", NULL, MODIFIABLE_CONFIG, server.hide_client_info, 1, NULL, NULL),
```

Comment:
I'm afraid setting this on by default. We redis developers) usually don't care about the key name and value in the crash report (users might want them though). But the args also have the sub commands and all sort of flags that completely change what the command does, losing that info could cause a rare non-reproducible crash even harder to figure out.

Reply:
I agree, we should disable it by default, and enable it in some redis.conf file